guides/common/modules/proc_setting-the-location-context-by-using-web-ui.adoc

```diff
@@ -0,0 +1,13 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="setting-the-location-context-by-using-web-ui"]
+= Setting the location context by using {ProjectWebUI}
+
+[role="_abstract"]
+A location context defines the location to use for a host and its associated resources.
+
+.Procedure
```

Comment:
I tweaked the procedure part to make sure it contains some actual steps. Minor changes only compared to the original content in its original location.